controllers/activities.js

```diff
@@ -28,6 +33,23 @@ function createAddNoteActivity({ previousValue, newValue, createdAt, twilioWorke
   };
 }
 
+function createAddReferralActivity({ previousValue, newValue, createdAt, twilioWorkerId }) {
```

Comment:
We've created an O(N^2) algorithm to populate an activity list.  For now we can live with it, but it seems rather unnecessary.  Could you create a jira ticket for future debt work that lists some ideas for how we could improve this?

Reply:
https://bugs.benetech.org/browse/CHI-436